android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -54,6 +58,19 @@ private static void throwEncodeException(JSONException e) {
     throw new RuntimeException("Could not encode JSON", e);
   }
 
+  public static String getApiKey(Context context){
+    int resourceId = context.getResources().getIdentifier("google_api_key", "string", context.getPackageName());
+    return resourceId != 0 ? context.getString(resourceId) : "";
```

Comment:
The way I thought this was going to be used, the google_api_key wasn't going to be present instead of being there but empty. When its the resource doesn't exist context.getString(R.string.google_api_key) causes an compile error.

Do you want me to change it, and always have a string resource named google_api_key?